src/ImageRunModal.jsx

```diff
@@ -175,6 +175,9 @@
     componentDidMount() {
         this._isMounted = true;
         this.onSearchTriggered(this.state.searchText);
+
+        if (this.props.prefill)
+            this.prefillModal();
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1270-20230428-110256-01fd85db-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)